portfolio/pom.xml

```diff
@@ -22,6 +22,11 @@
       <artifactId>gson</artifactId>
       <version>2.8.6</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>1.9.59</version>
+    </dependency>
```

Comment:
This is technically related to the next step (for Datastore), right?

Reply:
Yes it is.